content/index.html

```diff
@@ -0,0 +1,120 @@
+      <p>Elixir</p>
+    </a>
+  </div>
+  <div class="text-centered mr3">
+    <a class="tc link dim" href="https://dnsimple.com/api/node?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/node.svg">
+      <p>Node.js</p>
+    </a>
+  </div>
+  <div class="text-centered mr3">
+    <a class="tc link dim" href="https://dnsimple.com/api/csharp?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/dotnet.svg">
+      <p>.NET</p>
+    </a>
+  </div>
+  <div class="text-centered mr3">
+    <a class="tc link dim" href="https://dnsimple.com/api/java?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/java.svg">
+      <p>Java</p>
+    </a>
+  </div>
+  <div class="text-centered mr3">
+    <a class="tc link dim" href="https://dnsimple.com/api/php?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/php.svg">
+      <p>PHP</p>
+    </a>
+  </div>
+  <div class="text-centered mr3">
+    <a class="tc link dim" href="https://dnsimple.com/api/python?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/python.svg">
+      <p>Python</p>
+    </a>
+  </div>
+  <div class="text-centered">
+    <a class="tc link dim" href="https://dnsimple.com/api/rust?utm_source=developer&utm_medium=web&utm_content=cta-lang" target="_blank">
+      <img class="h2 w3" src="/assets/images/rust.svg">
+      <p>Rust</p>
+    </a>
+  </div>
+</div>
+
+<div class="pv3 flex flex-wrap justify-between">
+  <div class="card pa3 w-100 w-30-l">
+    <h2>Getting started</h2>
+    <p>Understand the core concepts behind the API.</p>
+    <a href="/getting-started">Get started</a>
+  </div>
+  <div class="card pa3 w-100 w-30-l">
+    <h2>Use cases</h2>
+    <p>Short tutorials demonstrating real-world use cases.</p>
+    <a href="/use-cases">Read more</a>
+  </div>
+  <div class="card pa3 w-100 w-30-l">
+    <h2>Popular artices</h2>
+    <ul class="pl3">
+      <li><a href="/v2/zones/records/">Zone records</a></li>
+      <li><a href="/Sandbox">Sandbox environment</a></li>
+      <li><a href="/v2/registrar/">Registrar API</a></li>
+    </ul>
+  </div>
+</div>
+
+<div class="pv4">
+  <h2 class="ma0 pb2 mb2 bb bw1 divider">New Features</h2>
+  <a href="https://blog.dnsimple.com/2022/06/support-webauthn-api/?utm_source=developer&utm_medium=web&utm_content=new-features" class="link dim" target="_blank">
+    <div class="flex flex-wrap justify-between items-center">
+      <div class="w-100 w-50-ns">
+        <img src="/assets/images/features/two-step-verification.png" alt="Security keys" class="br3">
+      </div>
+      <div class="w-100 w-50-ns pa3-ns">
+        <h3>Get Stronger Account Protection with Web Authentication (WebAuthn) Support</h3>
```

Comment:
We need to pull these directly from the blog. 😁

Reply:
Yes, automation would be the next step. Right now, it's just an experiment. We'll be measuring if this generates any traffic in the next couple of months.

Reply:
I fixed the unused image and the spacing around the blog posts. Thank you!